ipyleaflet/leaflet.py

```diff
@@ -1032,6 +1032,11 @@ def _default_options(self):
     east = Float(def_loc[1], read_only=True).tag(sync=True)
     west = Float(def_loc[1], read_only=True).tag(sync=True)
 
+    bottom = Int(0, read_only=True).tag(sync=True)
+    top = Int(9007199254740991, read_only=True).tag(sync=True)
```

Comment:
Why this default value?

Reply:
It's the biggest `Number`. We need this initial value in order to play well with the `min`.